package.json

```diff
@@ -13,32 +13,42 @@
   },
   "author": "Auth0",
   "license": "MIT",
-  "devDependencies": {
-    "babel-register": "^6.16.3",
-    "chai": "^3.5.0",
-    "cross-env": "^3.1.2",
-    "eslint": "^3.9.1",
-    "eslint-config-airbnb": "^15.1.0",
-    "eslint-plugin-babel": "^3.3.0",
-    "eslint-plugin-import": "^2.0.1",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.1.0",
-    "expect": "^1.20.2",
-    "express": "^4.14.0",
-    "mocha": "^3.1.2",
-    "nyc": "^8.3.1",
-    "supertest": "^2.0.0"
-  },
   "dependencies": {
-    "auth0": "^2.9.0",
-    "auth0-extension-tools": "^0.14.0",
+    "ajv": "^6.5.2",
+    "auth0": "auth0/node-auth0#master",
```

Comment:
This should probably be a specific version rather than #master in the long run...

Reply:
Yep, it will be next week when the new version is released. Used master for development in the meantime.